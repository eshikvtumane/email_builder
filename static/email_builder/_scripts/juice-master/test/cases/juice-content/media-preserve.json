{
    "url": "./",
    "removeStyleTags": true,
    "preserveMediaQueries": true
}