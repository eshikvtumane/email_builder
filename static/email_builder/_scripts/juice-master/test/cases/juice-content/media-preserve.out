<html>
<head>
    <style>
@media screen {
  a {
    color: blue !important;
    background: red;
  }
  @font-face {
    font-family: 'Arial2';
  }
}
</style>
</head>
<body>
    <p style="color: blue;">hi</p>
</body>
</html>
